sdk/python/feast/feature_table.py

```diff
@@ -64,6 +66,11 @@ def __init__(
         self._created_timestamp: Optional[Timestamp] = None
         self._last_updated_timestamp: Optional[Timestamp] = None
 
+        stack = inspect.stack()
+        # Get two levels up from current, to ignore usage.py
+        previous_stack_frame = stack[2]
+        self.defined_in = previous_stack_frame.filename
```

Comment:
Sounds like missing one `get_calling_file_name()` here

Reply:
oh yeah missed this